### ISSUE TITLE
Reject overly old overpass responses

### ISSUE THREAD

**Issue by matkoniecz** [westnordost/StreetComplete#1530]
In some cases Overpass instance may return very old data, in some cases database lag may reach months or years.

> >  not sure about current status but some time ago their databases had significant lag in updating.

> That's the reason why you should always analyse the contents in \<meta osm_base="..."\> and reject any data that is too old.

https://github.com/westnordost/StreetComplete/issues/1523

EDIT: fix \< \> escaping in quote

**Comment by westnordost** [comment]
Does Overpass report the age of the data?

**Comment by matkoniecz** [comment]
It seems so - see data of http://overpass-turbo.eu/s/LJo 

```
{
  "version": 0.6,
  "generator": "Overpass API 0.7.55.7 8b86ff77",
  "osm3s": {
    "timestamp_osm_base": "2019-08-22T14:20:02Z",
    "timestamp_areas_base": "2019-08-22T14:00:02Z",
    "copyright": "The data included in this document is from www.openstreetmap.org. The data is made available under ODbL."
  },
```

http://overpass-turbo.eu/s/LJp has

```
<meta osm_base="2019-08-22T14:21:02Z" areas="2019-08-22T14:00:02Z"/>
```

**Comment by westnordost** [comment]
Closing this as this should be implemented as part of #1523
